filesystem/apps/shell/main.rs

```diff
@@ -282,9 +282,9 @@ impl<'a> Command<'a> {
                     println!("URL: {:?}", file.path());
 
                     let string: String = args.iter()
-                        .skip(2)
-                        .fold(String::new(), |s, arg| s + " " + arg)
-                        + "\r\n\r\n";
+                                             .skip(2)
+                                             .fold(String::new(), |s, arg| s + " " + arg) +
+                                         "\r\n\r\n";
```

Comment:
This line is not indented consistently and I think it would look better to have the `+` in front of the this expression.